src/db/models/request.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Request = sequelize.define('Request', {
+    userEmail: DataTypes.STRING,
```

Comment:
This should be a foreign key referencing to a row in the `Users` table

Reply:
Here's the implementation, I have made `userEmail` the foreignKey in request table

<img width="458" alt="Screen Shot 2020-02-20 at 15 30 19" src="https://user-images.githubusercontent.com/48021034/74938229-469d5e00-53f6-11ea-8f4a-ee7782ee512b.png">